src/legacy/components/LoadingScreen.tsx

```diff
@@ -159,7 +160,7 @@ class LoadingScreen extends Component<any, any> {
     // Load the RPC config from pastel.conf file
     const pastelLocation = locatePastelConf()
     let confValues
-
+    await updateDefaultPastelConfig(pastelLocation)
```

Comment:
We need to do try/catch here please. Don't forget that we have a specific way of logging the error: `console.warn(<class> <function> error: ${error.message})`

Reply:
I have  updated for this one